PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRegister.swift

```diff
@@ -99,34 +98,18 @@ public class PrebidMobilePluginRegister: NSObject {
         }
     }
     
-    /// Returns the list of available renderers for the given ad unit for RTB request
-    public func getRTBListOfRenderersFor(for adUnitConfig: AdUnitConfig) -> [PrebidMobilePluginRenderer] {
-        queue.sync {
-            plugins
-                .values
-                .filter { renderer in
-                    adUnitConfig.adFormats.contains { format in
-                        renderer.isSupportRendering(for: format)
-                    }
-                }
-        }
-    }
-    
-    /// Returns the registered renderer according to the preferred renderer name in the bid response
-    /// If no preferred renderer is found, it returns PrebidRenderer to perform default behavior
-    /// Once bid is win we want to resolve the best PluginRenderer candidate to render the ad
-    public func getPluginForPreferredRenderer(
-        bid: Bid,
-        isInterstitial: Bool
-    ) -> PrebidMobilePluginRenderer {
-        if let preferredRendererName = bid.pluginRendererName,
-           let preferredPlugin = getPluginRenderer(for: preferredRendererName),
-           preferredPlugin.version == bid.pluginRendererVersion,
-           preferredPlugin.isSupportRendering(for: bid.adFormat) {
-            return preferredPlugin
-        } else {
-            return isInterstitial ? defaultInterstitialRenderer : defaultAdViewRenderer
+    /// Returns the registered renderer according to the preferred renderer name in the bid response.
+    /// If no preferred renderer is found, it returns PrebidRenderer to perform default behavior.
+    /// Once bid is win we want to resolve the best PluginRenderer candidate to render the ad.
+    public func getPluginForPreferredRenderer(bid: Bid) -> PrebidMobilePluginRenderer {
+        guard let preferredRendererName = bid.pluginRendererName,
+              let preferredPlugin = getPluginRenderer(for: preferredRendererName),
+              preferredPlugin.version == bid.pluginRendererVersion
+        else {
+            return defaultRenderer
         }
+        
+        return preferredPlugin
     }
     
     public func getAllPlugins() -> [PrebidMobilePluginRenderer] {
```

Comment:
Yes, we should register the default renderer in the list. It will be a good design approach. However, I'm not sure if SDK sends any signals indicating the presence of a default renderer. We'll double-check it.

Reply:
@jangirtony2009, we checked the Android behavior and @ValentinPostindustria says that the SDK doesn't send any info about the default rendered in the request. So could you clarify this part of your  comment 

> ... and sending it to the server

Tanks!